packages/angular/src/generators/stories/schema.json

```diff
@@ -6,14 +6,15 @@
   "cli": "nx",
   "type": "object",
   "properties": {
-    "name": {
+    "project": {
       "type": "string",
-      "description": "The name of the project.",
+      "aliases": ["name", "projectName"],
+      "description": "Project for which to generate stories.",
       "$default": {
-        "$source": "argv",
+        "$source": "projectName",
         "index": 0
       },
-      "x-prompt": "What's the name of the project for which you want to generate stories?"
+      "x-prompt": "For which project do you want to generate stories?"
```

Comment:
I already did the change for all generators in the angular plugin in a different PR.

Also, please note that we don't want to be changing the names of the properties if we can avoid it, it would break anyone invoking the generator programmatically (e.g. inside their own generators). We don't want to change the `$source` either from `argv`, because then we lose the ability to pass the project name as the first positional parameter. In those cases, it's better to set the `"x-dropdown": "projects"` option.

Reply:
Yep, I know. I just resolved the merge conflicts!! hehe! I added everything now. The dropdown, and all. 

That's why I added the aliases, to avoid breaking changes. But I wanted to change the `name` to `project` to be consistent across our repo!